### PR TITLE
Enable win build

### PR DESCRIPTION
**Pull request by ajaypnair09**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ajaypnair09** [comment]
@conda-forge/help-java Hello Experts, Here I would like to create new plantuml package for windows. But windows build is failing here. Can you please help me with suggesting where it is going wrong and how to fix it.

**Comment by bollwyvl** [comment]
Does this work locally?

**Comment by ajaypnair09** [comment]
@conda-forge-admin please rerender

**Comment by ajaypnair09** [comment]
> Does this work locally?

Yes. bld.bat is tested locally. It works fine

**Comment by ajaypnair09** [comment]
@conda-forge-admin , @bollwyvl Kindly review and merge if all fine

**Comment by bollwyvl** [comment]
Thanks for sticking with it. A few more things about error checking and the wrapper script,  and we should be good to go.

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,22 +1,8 @@
...
-
-echo java -Xmx500M -jar %LIBRARY_LIB%\plantuml-pdf.jar %%* > %LIBRARY_BIN%\plantuml.cmd
-echo IF %%ERRORLEVEL%% NEQ 0 EXIT /B %%ERRORLEVEL%% >> %LIBRARY_BIN%\plantuml.cmd
-
-type %LIBRARY_BIN%\plantuml.cmd
-
-:error
-echo Failed with error #%errorlevel%.
-exit /b %errorlevel%
+echo java -Xmx500M -jar ^%%CONDA_PREFIX^%%\Library\lib\plantuml.jar "$@" %%*> %LIBRARY_BIN%\plantuml.bat
```
[the remark this answers is not in the document]
Comment (ajaypnair09) [reply recipe/bld.bat]: Hello @bollwyvl, Updated. Please have a look again

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
Thanks!

**Pull request merged**